lib/src/airship_flutter.dart

```diff
@@ -215,6 +215,10 @@ class Airship {
     return AttributeEditor('editAttributes', _channel);
   }
 
+  static AttributeEditor editNamedUserAttributes() {
```

Comment:
could we deprecate `editAttributes` and add new `editChannelAttributes`

Reply:
done